simple_install/install.bat.in

```diff
@@ -0,0 +1,180 @@
+@echo off
+
+@rem This script requires the user to install Python 3.9 or higher. All other
+@rem requirements are downloaded as needed.
+
+@rem change to the script's directory
+PUSHD "%~dp0"
+
+set "no_cache_dir=--no-cache-dir"
+if "%1" == "use-cache" (
+    set "no_cache_dir="
+)
+
+exit /b
+
+echo ***** Installing InvokeAI.. *****
+@rem Config
+
+@rem this should be changed to the tagged release!
+@rem set INVOKE_AI_SRC=https://github.com/invoke-ai/InvokeAI/archive/main.zip
+set INVOKE_AI_SRC=https://github.com/invoke-ai/InvokeAI/archive/lstein-simple-install.zip
+set INSTRUCTIONS='<insert URL here>'
+set TROUBLESHOOTING='<insert URL here>'
+set MINIMUM_PYTHON_VERSION=3.9.0
+
+@rem ---------------------------- check Python version ---------------
+call python --version >.tmp1 2>.tmp2
+if "%ERRORLEVEL%" neq "0" (
+   echo Please install Python 3.9 or higher. See %INSTRUCTIONS% for details.
+   exit /b
+
+where /q virtualenv
+if %errorlevel% ne 0 (
+   echo Installing virtualenv support.
+   call pip install virtualenv
+   if %errorlevel% ne 0 (
+      echo The pip program failed to install virtualenv support. Please see %TROUBLESHOOTING% for help.
+      exit /b
```

Comment:
This isn't necessary on Windows as `venv` module comes preinstalled
```suggestion
```

Reply:
I don't actually know which of the `venv` or `virtualenv` modules to prefer. They seem to do the same thing, but `virtualenv` copies python into its environment, which I guess is better?

Reply:
`venv` is the preferred method since python3.3, because it's integrated into standard library. Though on Ubuntu it still has to be installed separately. We can point it out in the docs - I don't think there's a way around that. 

For copying the python binary, `venv` has the `--copies` flag, but I don't think I've ever had a problem just letting it use the symlinks, which is the default on Linux.

Reply:
> Though on Ubuntu it still has to be installed separately

Ubuntu is making me very stabby.